src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -1062,4 +1068,48 @@ public static Font getTooltipFont(@Nullable Font forcedFont, @Nonnull ItemStack
             return Boolean.compare(vanillaTwo, vanillaOne);
         };
     }
+
+    public static void createWorldConfirmationScreen(
+            LevelStorageSource save, String worldName, boolean creatingWorld,
+            Function4<LevelStorageSource.LevelStorageAccess, RegistryAccess.RegistryHolder, ResourceManager, DataPackConfig, WorldData> f4,
+            Function<Function4<LevelStorageSource.LevelStorageAccess, RegistryAccess.RegistryHolder, ResourceManager, DataPackConfig, WorldData>, Runnable> runAfter)
+    {
+        Component title = new TranslatableComponent("selectWorld.backupQuestion.experimental");
+        Component msg = new TranslatableComponent("forge.selectWorld.backupWarning.experimental");
```

Comment:
I just had an idea. Instead of supplanting the vanilla `selectWorld.backupWarning.experimental` translation with our own version that includes a sentence in the middle of the text, we can instead append our text after the vanilla text:
```json5
{
	// ... other translations
	"forge.selectWorld.backupWarning.experimental.additional": "This message will not show again for this world."
}
```
```suggestion
        Component msg = new TranslatableComponent("selectWorld.backupWarning.experimental")
                .append("\n\n")
                .append(new TranslatableComponent("forge.selectWorld.backupWarning.experimental.additional"));
```

This looks like the following (and I have confirmed that the Narrator reads it correctly, for accessibility):
![Experimental settings confirmation screen with additional text of "This message will not show again for this world."](https://user-images.githubusercontent.com/21304337/149656538-8da75713-e8b9-4f1b-acaf-140e8c8d4032.png)

The benefits of this approach is that we only need to provide translations for the text we add rather than re-translating the whole vanilla text with our included text, and this emphasizes to the user that the screen will not show again afterwards when they proceed.